### PR TITLE
Compare submission quality

### PR DESCRIPTION
**Pull request by hajiler**
Might do: Add functionality for editing fields? Fix typos etc

### CODE REVIEW

File: website/src/main/java/com/google/step/data/ClassHandler.java [hunk 1]
```diff
@@ -40,4 +40,10 @@ default ClassifyTextRequest convertDocToRequest(Document doc) {
   default Document convertStringToDoc(String text) {
     return Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT).build();
   }
+
+  public static ClassifyTextResponse getTestCategory() {
+    ClassificationCategory c = ClassificationCategory.newBuilder().build()
+        .toBuilder().setName("/Fake/Fake1/Fake2/Fake3").build();
```
Comment (tonypan2000) [inline website/src/main/java/com/google/step/data/ClassHandler.java:46]: Please remove test categories for final product, or separate test/prod environments

File: website/src/main/java/com/google/step/data/CloudSQLManager.java [hunk 2]
```diff
@@ -79,12 +79,21 @@ public void drop(String tableName) {
...
     String query = String.format("SELECT DISTINCT %s FROM %s%s", values, tableName, where);
     Statement stmt = this.conn.createStatement();
     return stmt.executeQuery(query);
   }
 
+  //Get all GN4P orgs similar to org
+  public ResultSet getPossibleComparisons(String tableName, OrganizationInfo org) throws SQLException {
+    List<String> attributes = Arrays.asList(
+        String.format("name LIKE %s", "'%" + org.getName() + "%'"),
+        String.format("link LIKE %s", "'%" + org.getLink() + "%'"),
```
Comment (tonypan2000) [inline website/src/main/java/com/google/step/data/CloudSQLManager.java:91]: Optional: link likely will never be the same as a keyword, so I don't know how necessary this is, but ok to keep as it is.

File: website/src/main/java/com/google/step/data/CloudSQLManager.java [hunk 3]
```diff
@@ -168,6 +177,31 @@ public void executeStatement(String query) throws SQLException {
     stmt.executeUpdate(query);
   }
 
+  public ResultSet getFirstUploadOrg(String tableName) throws SQLException {
+    String query = String.format("SELECT * FROM %s ORDER BY id LIMIT 0, 1", tableName);
```
Comment (tonypan2000) [inline website/src/main/java/com/google/step/data/CloudSQLManager.java]: if you are just getting the first element, you don't need to specify the offset as 0, so "LIMIT 1" is sufficient

File: website/src/main/java/com/google/step/data/ClassHandler.java [hunk 4]
```diff
@@ -40,4 +40,10 @@ default ClassifyTextRequest convertDocToRequest(Document doc) {
...
+    return ClassifyTextResponse.newBuilder().addCategories(c).build();
+  }
 }
```
Comment (tonypan2000) [inline website/src/main/java/com/google/step/data/ClassHandler.java]: nit: end file with one empty line

**Review by tonypan2000 — APPROVED** [review]
LGTM but please fix style and do more testing

File: website/src/main/webapp/admin.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+          </section>
+        </div>
+      </header>
+    </div>
+
+    <h3 id="results-title">Submission</h3>
+    <div id="submission"></div>
+    <div id="comparisons"></div>
+  </body>
+</html>
```
Comment (tonypan2000) [inline website/src/main/webapp/admin.html]: end file on empty line

File: website/src/main/java/com/google/step/data/ClassHandler.java (hunk 1 above)
Comment (hajiler) [reply website/src/main/java/com/google/step/data/ClassHandler.java:46]: A separate environment for production is coming soon

**Pull request merged**